hermes/filters.py

```diff
@@ -56,6 +58,30 @@ def filter_uuid(self, queryset, name, value):
     def filter_referencing_uuid(self, queryset, name, value):
         return queryset.filter(data__references__contains=[{'citation': value}])
 
+    def filter_search(self, queryset, name, value):
+        query_terms = []
+        for i, term in enumerate(value.split('"')):
+            if term.strip():
+                if i % 2 == 0:
+                    for subterm in term.split(' '):
+                        if subterm.strip():
+                            query_terms.append(subterm.strip())
+                else:
+                    # Assume this piece is within double quotes, so don't split it
+                    query_terms.append(term.strip())
+
+        aggregate_keyword_query = Q()  # empty Q-object doesn't even add WHERE clause to SQL
+        for term in query_terms:
+            aggregate_keyword_query = aggregate_keyword_query | Q(title__icontains=term)
```

Comment:
Should we search a creation date/range?
Or are we saving that for an advanced search function?

Reply:
yeah I left out date searches in here because its a can of worms with syntax of greater than / less than dates and stuff. I think its better handled by dedicated UI elements to select dates and filter by that specifically.

Reply:
:+1: